dspace-api/src/main/java/org/dspace/content/dao/impl/ItemDAOImpl.java

```diff
@@ -138,6 +138,7 @@ protected ItemDAOImpl()
     @Override
     public Iterator<Item> findByMetadataQuery(Context context, List<List<MetadataField>> listFieldList, List<String> query_op, List<String> query_val, List<UUID> collectionUuids, String regexClause, int offset, int limit) throws SQLException {
     	Criteria criteria = createCriteria(context, Item.class, "item");
+        criteria.add(Property.forName("inArchive").eq(Boolean.TRUE));
```

Comment:
While I'm not against this change, it seems like we need to do one of two things...

1. Either, we should consider changing `findByMetadataQuery()` to accept an "in_archive" parameter, in the same way that `findAll()` and `findByMetadataField()` do in this same class.
2. OR, we need to add Javadocs to `ItemDAO` interface for `findByMetadataQuery()` to make it clear that it *only* searches Items that are "in_archive=true". 

I lean slightly towards option 1, since it's more in line with other methods in this class. However, if that change proves to be very complex, then option 2 is OK.

Reply:
I've pushed up a commit to change findByMetadataQuery() as per option 1. it was not a big or complicated change as FilteredItemsResource is the only class currently using this method.

I made the change in ItemDAO and ItemService interfaces, and their implementations, and set FilteredItemsResource to pass `true` when calling findByMetadataQuery()

Hope that's OK Terry! Just revert my last commit if you had another idea in mind. I'll approve this now as I'm also a +1